http://bugs.gentoo.org/show_bug.cgi?id=124657
https://bugs.freedesktop.org/show_bug.cgi?id=5635

===================================================================
RCS file: /srv/anoncvs.freedesktop.org/cvs/xlibs/xkbdesc/types/level5,v
rcsdiff: /srv/anoncvs.freedesktop.org/cvs/xlibs/xkbdesc/types/level5,v: warning: Unknown phrases like `commitid ...;' are present.
retrieving revision 1.2
retrieving revision 1.3
diff -u -r1.2 -r1.3
--- xkbdesc.orig/types/level5	2005/10/17 00:40:47	1.2
+++ xkbdesc/types/level5	2006/01/22 17:49:45	1.3
@@ -6,8 +6,7 @@
     // Defines style a with four level shift. It is similar to three_level
     // type used for iso9995 but it lets the shift key behave "normally".
 
-    virtual_modifiers LevelThree;
-    virtual_modifiers LevelFive;
+    virtual_modifiers LevelThree,ScrollLock,LevelFive;
 
     type "EIGHT_LEVEL" {
         modifiers = Shift+LevelThree+LevelFive;
